Dockerfile

```diff
@@ -99,6 +103,7 @@ COPY --from=generate-build /api/machine/machine.pb.go /pkg/machinery/api/machine
 COPY --from=generate-build /api/time/time.pb.go /pkg/machinery/api/time/
 COPY --from=generate-build /api/network/network.pb.go /pkg/machinery/api/network/
 COPY --from=generate-build /api/cluster/cluster.pb.go /pkg/machinery/api/cluster/
+COPY --from=generate-build /pkg/machinery/config/types/v1alpha1 /pkg/machinery/config/types/v1alpha1
```

Comment:
is there a way to copy here only generated files (without source files), so that we never clobber local edits?